src/video/wayland/SDL_waylandevents.c

```diff
@@ -68,9 +69,14 @@
 // Weston uses a ratio of 10 units per scroll tick
 #define WAYLAND_WHEEL_AXIS_UNIT 10
 
-// xkbcommon as of 1.4.1 doesn't have a name macro for the mode key
-#ifndef XKB_MOD_NAME_MODE
-#define XKB_MOD_NAME_MODE "Mod5"
+// "Mod5" is typically level 3 shift, which SDL calls SDL_KMOD_MODE (AltGr).
```

Comment:
Mod5 is level 3 and Mod3 is level 5? Is that right?

Reply:
Yep. It's a bit counter intuitive, but that's how it's mapped in the default Xkb config. Actually, Mod5 is actually "mode switch", which is _activated_ by level 3 shift (AltGr). I'll fix the comment so it sounds less confusing.

Reply:
Actually, looking again, Mod5 is actually defined as "ISO_Level3_Shift":
```
shift       Shift_L (0x32),  Shift_R (0x3e)
lock        Caps_Lock (0x42)
control     Control_L (0x25),  Control_R (0x69)
mod1        Alt_L (0x40),  Alt_R (0x6c),  Alt_L (0xcc),  Meta_L (0xcd)
mod2        Num_Lock (0x4d)
mod3        ISO_Level5_Shift (0xcb)
mod4        Super_L (0x85),  Super_R (0x86),  Super_L (0xce),  Hyper_L (0xcf)
mod5        ISO_Level3_Shift (0x5c)
```

Either way, it is correct, if counterintuitive, and the comments now note it as such.